src/after-compile.ts

```diff
@@ -203,16 +204,13 @@ function provideErrorsToWebpack(
       continue;
     }
 
-    const errors =
-      program === undefined
-        ? [
-            ...languageService!.getSyntacticDiagnostics(filePath),
-            ...languageService!.getSemanticDiagnostics(filePath)
-          ]
```

Comment:
Just out of curiosity, can you recall why we used to alternate between `languageService` and `program` here?

Reply:
To clarify, these lines weren't the issue, I just updated them for consistency with the changes above, which were the issue.

`languageService.get*Diagnostics()` essentially just calls its internal `program.get*Diagnostics()`. Based on loader options (and I honestly can't remember what options produce what scenarios), we either have a program directly (`instance.program`) or a language service which has a program under the hood (`instance.languageService.getProgram()`). Before, we were branching here in this function based on where the underlying program is. But, we already have a util—`ensureProgram`—that does that branching.

TL;DR: these lines are basically equivalent because of `ensureProgram`; the lines above _should_ have been looking at both `program` and `languageService` but were only looking at `program`.